templates/rails/files/app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```

Comment:
Why do we need this file?

Reply:
It was auto-generated by the Rails setup script.  I have no idea what it does, honestly.